test/api-documentation.test.js

```diff
@@ -571,4 +589,237 @@ describe('<api-documentation>', function() {
       assert.notOk(node, 'method is not endered');
     });
   });
+
+  [
+    ['Compact model', true],
+    ['Regular model', false]
+  ].forEach(([name, compact]) => {
+    describe(name, () => {
+      describe('Server selection', () => {
+        let element;
+        let amf;
+
+        describe('in narrow mode', () => {
+          beforeEach(async () => {
+            amf = await AmfLoader.load(null, compact);
+            element = await narrowFixture(
+              amf,
+              'method',
+              html`
```

Comment:
Please, don't do this like this. Define the DOM tree in the template function. There's no point of parameterizing this function if you are using this once. If that makes you feel better declare this fixture in `Server selection` describe block :)

Reply:
great!